src/tests/unittests/WireTests.cpp

```diff
@@ -408,6 +408,39 @@ TEST_F(WireTests, StructureOfValuesArgument) {
     FlushClient();
 }
 
+// Test that the wire is able to send structures that contain objects
+TEST_F(WireTests, StructureOfObjectsArgument) {
```

Comment:
Thank you for adding this test! nit: I think it could be names "StructureOfObjectArrayArgument"